<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkStdMultiWidgetEditorPreferencePage</class>
 <widget class="QWidget" name="QmitkStdMultiWidgetEditorPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>External Programs</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="m_EnableFlexibleZooming">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Use constraint zooming and padding</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_ShowLevelWindowWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Show level/window widget</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="m_PACSLikeMouseMode">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Use PACS like mouse interaction (select left mouse button action)</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gradient background (3D renderwindow)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>First color:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="m_ColorButton1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Second color:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="m_ColorButton2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="m_ResetButton">
     <property name="text">
      <string>Reset Colors</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Rendering Mode*</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="m_RenderingMode">
     <item>
      <property name="text">
       <string>Standard Rendering</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enable Multisampling (Antialiasing)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enable Depth Peeling</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>* Changes require restart of MITK.
 Depth Peeling is only supported by Windows.
 For other OS, use Standard Rendering and enable
 the property 'Depth Sorting' in the property list of the surface data node.</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Render windows</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="m_RenderWindowChooser">
     <item>
      <property name="text">
       <string>stdmulti.widget1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stdmulti.widget2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stdmulti.widget3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stdmulti.widget4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="m_RenderWindowDecorationColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="m_RenderWindowDecorationText"/>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Crosshair gap size</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QSpinBox" name="m_CrosshairGapSize">
     <property name="value">
      <number>32</number>
     </property>
    </widget>
   </item>
   </widget>
  </layout>
 <resources/>
 <connections/>
</ui>
